tests/ui/allow_attribute.rs

```diff
@@ -0,0 +1,18 @@
+// run-rustfix
+#![allow(unused)]
+#![warn(clippy::allow_attribute)]
+#![feature(lint_reasons)]
+
+fn main() {}
+
+// Using clippy::needless_borrow just as a placeholder, it isn't relevant.
```

Comment:
What do you mean with this comment? :sweat_smile:

Reply:
T3 and T4 have attributes `#[deny(clippy::needless_borrow)]` and `#[warn(clippy::needless_borrow)]` to check that the lint doesn't change anything except `allow`. So it means that the lint `needless_borrow` is used but isn't meaningful and could be replaced.